packages/runtime/src/prop-controllers/descriptors.ts

```diff
@@ -205,6 +205,7 @@ type BackgroundImageV0 = {
   repeat?: BackgroundImageRepeat
   opacity?: number
   parallax?: number
+  priority?: boolean
```

Comment:
Where would this value be coming from? Are we also updating the backgrounds panel to add a priority?

Reply:
Yes. I figured we'd merge this first to expect the priority panel before we add it to the UI.